lib/mix/tasks/phx.gen.html.ex

```diff
@@ -195,9 +195,9 @@ defmodule Mix.Tasks.Phx.Gen.Html do
         ~s"""
         <.input field={{f, #{inspect(key)}}} type="select" label="#{label(key)}">
           <:option>Choose a value</:option>
-          <%= for value <- Ecto.Enum.values(#{inspect(schema.module)}, #{inspect(key)}) do %>
```

Comment:
We want the for comprehension here to build a template with an individual `<:option>...</:option>` per value. Can you make the change?

Reply:
@chrismccord could you elaborate? There's a `:for` on the new code and we get the expected individual option per value.

Reply:
ah sorry, I thought this was the .exs template, where we want invidiual `<:option>`'s, but I see the sis the injected body of the enum